### ISSUE TITLE
Not correct parse srcset

### ISSUE THREAD

**Issue by Grafs** [s0ph1e/node-website-scraper#34]
Hi,
Not correct parsing srcset in img tag
Example:
```
<img src="http://example.com/prev4-45x45.jpg" srcset="http://example.com/prev4-150x150.jpg 150w, http://example.com/prev4-45x45.jpg 45w" sizes="(max-width: 45px) 100vw, 45px" width="45" height="45">
```
Please, make option for convert all other links from relative to absolute.
Thank you

**Comment by s0ph1e** [comment]
Hi
Thanks for reporting the issue
I'll try to fix it asap

**Comment by s0ph1e** [comment]
Fixed in version 0.3.4
